---
title: "spliceJAC: transition genes and state-specific gene regulation from single-cell transcriptome data"
collection: publications
permalink: /publication/spliceJAC
excerpt: 'Using attractor Jacobian to infer cell type stability with mRNA splicing dynamics with scRNA-seq data'
date: 2021-9-23
venue: 'Nature Communications'
paperurl: 'https://doi.org/10.15252/msb.202211176'
citation: 'Federico Bocci, Peijie Zhou and Qing Nie. spliceJAC: transition genes and state-specific gene regulation from single-cell transcriptome data. https://www.embopress.org/doi/full/10.15252/msb.202211176'
---
**Abstract:** Extracting dynamical information from single-cell transcriptomics is a novel task with the promise to advance our understanding of cell state transition and interactions between genes. Yet, theory-oriented, bottom-up approaches that consider differences among cell states are largely lacking. Here, we present spliceJAC, a method to quantify the multivariate mRNA splicing from single-cell RNA sequencing (scRNA-seq). spliceJAC utilizes the unspliced and spliced mRNA count matrices to constructs cell state-specific gene–gene regulatory interactions and applies stability analysis to predict putative driver genes critical to the transitions between cell states. By applying spliceJAC to biological systems including pancreas endothelium development and epithelial–mesenchymal transition (EMT) in A549 lung cancer cells, we predict genes that serve specific signaling roles in different cell states, recover important differentially expressed genes in agreement with pre-existing analysis, and predict new transition genes that are either exclusive or shared between different cell state transitions.

[Federico Bocci, Peijie Zhou and Qing Nie. spliceJAC: transition genes and state-specific gene regulation from single-cell transcriptome data. Molecular Systems Biology 18, e11176 (2022).](https://www.embopress.org/doi/full/10.15252/msb.202211176)
